packages/main/src/ColorPalette.ts

```diff
@@ -215,6 +224,52 @@ class ColorPalette extends UI5Element {
 		}
 	}
 
+	onAfterRendering() {
+		if (this._shouldFocusRecentColors && this.hasRecentColors) {
+			this.recentColorsElements[0].selected = true;
+			this.recentColorsElements[0].focus();
+
+			this._shouldFocusRecentColors = false;
+		}
+
+		this._ensureSingleSelectionOrDeselectAll();
```

Comment:
Our colors slot is defined with invalidateOnChildChange so this this call here will trigger invalidation.